pages/linux/pacman-s.md

```diff
@@ -0,0 +1,7 @@
+# pacman-s
```

Comment:
```suggestion
# pacman -S
```
@sebastiaanspeck What do you think, should the title reflect the actual command and differ from the filename in this case?